### DISCUSSION TITLE
Custom FormStep template all properties not defined: Property "visible" was accessed during render but is not defined on instance.

### DISCUSSION THREAD

**Discussion by ptitb** [vueform/vueform#339] [Questions] [answered]
### Environment

``` 
"nuxt": "^3.12.4",
"vue": "^3.4.38",
"@vueform/nuxt": "^1.7.0"
```

### Reproduction

I created a custom template and copied FormStep.vue as described ([https://vueform.com/docs/styles-and-layout#replacing-templates](https://vueform.com/docs/styles-and-layout#replacing-templates)) to a custom template. 

I added the template in the following way:
```javascript
<Vueform :templates="{ FormStep }">
...
<script setup>
import FormStep from '/components/CustomFormStep.vue'
```
It looks like every property used in the template is missing and not being used in my custom template.

How can I fix this?

Thanks!

**Comment by adamberecz** [comment, marked as answer]
The issue here is that `FormStep` is also exported as a component not just a variable that you correctly pass to `templates`.

The solution could be to rename it to eg. `CustomFormStep`:
```vue
<Vueform :templates="{ FormStep: CustomFormStep }">
...
<script setup>
import CustomFormStep from '/components/CustomFormStep.vue'
```

https://stackblitz.com/edit/github-13wu3m?file=src%2Fcomponents%2FFormStep.vue,src%2FApp.vue

**Reply by ptitb** [reply]
Thanks Adam!
